include/fem-dem/gls_vans.h

```diff
@@ -206,7 +206,6 @@ class GLSVANSSolver : public GLSNavierStokesSolver<dim>
 
   CFDDEMSimulationParameters<dim> cfd_dem_simulation_parameters;
 
-private:
   void
   read_dem();
```

Comment:
No it should still be private. It was a mistake. I had to make all other void_fraction variables such as the nodal_void_fraction_relevant .... protected instead of private since I needed to access them in the cad_dem_coupling.cc to be able to write and read the checkpoint. I made the read_dem private again.